parm/config/gfs/config.ocnanal

```diff
@@ -6,21 +6,15 @@
 echo "BEGIN: config.ocnanal"
 
 export OBS_YAML_DIR="${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/config"
-export OBS_LIST=@SOCA_OBS_LIST@
-export OBS_YAML="${OBS_LIST}"
-export FV3JEDI_STAGE_YAML="${HOMEgfs}/sorc/gdas.cd/test/soca/testinput/dumy.yaml"
+export OBS_LIST=@SOCA_OBS_LIST@  # TODO(GA): doesn't look necessary as is to have
+export OBS_YAML="${OBS_LIST}"    #           OBS_LIST and OBS_YAML pick one or add logic
 export SOCA_INPUT_FIX_DIR=@SOCA_INPUT_FIX_DIR@
```

Comment:
This need not be this way. 
See above comment.
If there is a dependence on the number of levels, we can make a case statement here and cover all cases.